src/algorithms/lbfgs.h

```diff
@@ -0,0 +1,89 @@
+#pragma once
+
+#include <RcppArmadillo.h>
+#include "../families/family.h"
+#include "wolfe_line_search.h"
+
+using namespace Rcpp;
+using namespace arma;
+
+// // LBFGS Algorithm.
+// // Solves argmin_z { f(z) + rho/2*||z-u||^2 }
+template <typename T>
+mat Family::lbfgs(const T& x, const mat&y, const double rho, const mat& u){
```

Comment:
in the current code style we add a linebreak before `{`s.
```suggestion
mat Family::lbfgs(const T& x, const mat&y, const double rho, const mat& u)
{
```